praetorian_cli/sdk/test/ui/test_aegis_schedule.py

```
@@ -0,0 +1,552 @@
+
+# --- add_schedule tests ---
+
+class TestAddSchedule:
+    def test_add_schedule_success(self, monkeypatch):
+        """Happy path: add_schedule creates a schedule with correct params."""
+        menu = ScheduleMenu(schedules=SAMPLE_SCHEDULES)
+
+        # Mock capability picker to return a capability name
+        monkeypatch.setattr(
+            'praetorian_cli.ui.aegis.commands.schedule._interactive_capability_picker',
+            lambda m: 'windows-smb-snaffler'
```

Ruff (0.14.14)

[warning] Unused lambda argument: `m`

(ARG005)